tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/core/Ones.java

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.tensorflow.op.core;
+
+import org.tensorflow.DataType;
+import org.tensorflow.Operand;
+import org.tensorflow.Operation;
+import org.tensorflow.Output;
+import org.tensorflow.op.Op;
+import org.tensorflow.op.Scope;
+import org.tensorflow.op.annotation.Endpoint;
+import org.tensorflow.op.annotation.Operator;
+import org.tensorflow.op.dtypes.Cast;
+import org.tensorflow.types.TString;
+import org.tensorflow.types.family.TNumber;
+import org.tensorflow.types.family.TType;
+
+/**
+ * An operator creating a constant initialized with ones of the shape given by `dims`.
+ * 
+ * <p>For example, the following expression
+ * <pre>{@code tf.ones(tf.constant(shape), TFloat32.DTYPE)</pre>
```

Comment:
The code block is not closed (same thing below)

Reply:
I think this is due to a cc generation issue, as this is copied from Zeroes and the same issues are present there.